examples/gnrc_xbee_border_router/README.md

```diff
@@ -0,0 +1,117 @@
+# gnrc_xbee_border_router example
+
+## Requirements
+
+In order to setup a 6LoWPAN border router with XBee 802.15.4 on RIOT, you need a
+board that offers an IPv6 capable network interface (e.g. the `encx24j600`
+Ethernet chip) in addition to XBee. The example application in this folder
+assumes as a default to be run on an native platform using tap interface for
+ethernet and serial port (/dev/tty*) for communicating with XBee. However, it is
+feasible to run the example on any RIOT supported platform that is be equipped
+with an IPv6 capable network device. In this case, specify the target platform
+as `make BOARD=myplatform`.
+
+If XBee is connected to a UART port other than 0, change `XBEE_UART` variable
+in the Makefile. If the baudrate is other than 9600, configure it in
+`xbee_conf.h`.
+
+See `img/page_001.svg` for the organization of modules and devices.
```

Comment:
You can link images with the `![Caption](link)` syntax. With this notation, people who look at this page in the parsed version wouldn't likely know what you mean ;-)